.eslintignore

```diff
@@ -0,0 +1 @@
+/test/*
```

Comment:
Why? I mean, even tests _should_ follow the same guide-style. IMHO

Reply:
I follow the same guide-style in the tests, but I added the .eslintignore cause my editor do not recognize "it" and "describe" and when I see the code with red underline is very stressed for me!

Reply:
Oh, then you need to configure your .eslintrc to recognize _mocha_ calls instead, we'll figure out that later.